designs/black-parrot/tests/post.bash

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -x
+set -e
+# stdout must contain ''
+grep -q "All cores finished! Terminating" _execute/stdout.log
```

Comment:
I assume this means all cores finished successfully, and e.g. not crashed. Please consider if there is a stronger check you can do here that positively proves *correct* termination. If this is that it's fine.